velox/common/file/tests/FileTest.cpp

```diff
@@ -107,3 +108,51 @@ TEST(LocalFile, viaRegistry) {
   ASSERT_EQ(readFile->pread(0, 5, &buffer1), "snarf");
   lfs->remove(filename);
 }
+
+TEST(LocalFile, exist) {
+  filesystems::registerLocalFileSystem();
+  auto tempFile1 = ::exec::test::TempFilePath::create();
+  const auto& filename1 = tempFile1->path.c_str();
+  remove(filename1);
+  auto tempFile2 = ::exec::test::TempFilePath::create();
+  const auto& filename2 = tempFile2->path.c_str();
+  remove(filename2);
+  auto lfs = filesystems::getFileSystem(filename1, nullptr);
+  {
+    auto writeFile = lfs->openFileForWrite(filename1);
```

Comment:
We don't need to write anything to a file, do we? We can test exists on empty files.